### PR TITLE
Migrate Passenger model to User model (solves #308)

### PR DESCRIPTION
**Pull request by ngiersz**
- Use historical content of available models (in migration 0018) in app while migrating Passenger model to User model (instead of changing all previous migrations).
- Remove passengers field from Drive model (relation many to many with Passenger model). Add passenger field (foreign key to User model) instead.  
- Remove Passenger model and all its occurences.

### CODE REVIEW

**Comment by arturtamborski** [comment]
This looks really promising :1st_place_medal:  Please update the title & description to be more clear on the problem it solves.

**Comment by juliaanholcer** [comment]
@ngiersz We left unused import to Passenger model in populate_database.py and there are some tests that need to be fixed in test_drives.py. Will you continue?

File: backend/fleet_management/serializers.py [hunk 1]
```diff
@@ -59,28 +59,26 @@ class Meta:
 class DriveSerializer(serializers.ModelSerializer):
     driver = UserSerializer(read_only=True)
     car = CarSerializer()
-    passengers = PassengerSerializer(many=True)
+    passenger = PassengerSerializer()
```
[the remark this answers is not in the document]
Comment (magul) [reply backend/fleet_management/serializers.py]: Yes, creation of new drive causes `400` with:
```
{"passenger":{"nonFieldErrors":["Invalid data. Expected a dictionary, but got int."]}}
```

[the remark this answers is not in the document]
Comment (magul) [reply backend/fleet_management/serializers.py]: Actuall change to be applied on top of these all changes looks like:
```
diff --git a/backend/fleet_management/serializers.py b/backend/fleet_management/serializers.py
index 6467e92..2fb33b2 100644
--- a/backend/fleet_management/serializers.py
+++ b/backend/fleet_management/serializers.py
@@ -56,17 +56,26 @@ class ProjectSerializer(serializers.ModelSerializer):
         fields = ('title', 'description', 'id')
 
 
+class PassengersField(serializers.Field):
+
+    def to_representation(self, value):
+        return [PassengerSerializer(value).data]
+
+    def to_internal_value(self, data):
+        return data[0]
+
+
 class DriveSerializer(serializers.ModelSerializer):
     driver = UserSerializer(read_only=True)
     car = CarSerializer()
-    passenger = PassengerSerializer()
+    passengers = PassengersField(source="passenger")
     project = ProjectSerializer()
 
     class Meta:
         model = Drive
         fields = (
             'id',
-            'driver', 'car', 'passenger', 'project',
+            'driver', 'car', 'passengers', 'project',
             'date', 'start_mileage', 'end_mileage', 'description',
             'start_location', 'end_location', 'timestamp'
         )
```

**Pull request merged**